opacclient/opacapp/src/main/res/values/strings.xml

```diff
@@ -59,19 +59,13 @@
     <string name="notification_service">Expiring items</string>
     <string name="notification_service_desc">Notify, if an item is about to expire (no warranties given)</string>
     <string name="notification_service_wifionly">Check on WiFi only</string>
-    <string name="notification_service_wifionly_desc">Does not download information over mobile networks. (Therefore, no warnings without WiFi!)</string>
+    <string name="notification_service_wifionly_desc">Does not download information over mobile networks.</string>
```

Comment:
How about „You might miss some notifications if you go without WiFi for a longer time“?